From: Robert Schwebel <r.schwebel@pengutronix.de>
Subject: [xf86-video-apm] add sysroot support

The xf86-video-apm packet tries to include headers from
$(prefix)/include, which isn't correct. Usually, when building
distributions, packets are configured with --prefix=/usr and installed
with SYSROOTS=....

Signed-off-by: Erwin Rol <erwin@erwinrol.com>
Signed-off-by: Luotao Fu <l.fu@pengutronix.de>
Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 configure.ac |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xf86-video-apm-1.2.2/configure.ac
===================================================================
--- xf86-video-apm-1.2.2.orig/configure.ac
+++ xf86-video-apm-1.2.2/configure.ac
@@ -67,7 +67,7 @@ AC_SUBST([XF86RUSH_LIBS])
 AC_SUBST([XF86RUSH_CFLAGS])
 
 CFLAGS="$CFLAGS $XORG_CFLAGS $XF86RUSH_CFLAGS"' -I$(top_srcdir)/src'
-INCLUDES="$XORG_INCS -I${sdkdir} $XF86RUSH_INCS "'-I$(top_srcdir)/src -I$(prefix)/include'
+INCLUDES="$XORG_INCS -I${sdkdir} $XF86RUSH_INCS "'-I$(top_srcdir)/src '"-I${SYSROOT}"'$(prefix)/include'
 AC_SUBST([CFLAGS])
 AC_SUBST([INCLUDES])
 
